docs/navigation-parameters-reference.md

```diff
@@ -96,6 +96,20 @@ The navigation parameters allow you to configure **global** navigation settings
 - **description**: renders all views in new frames. This setting overrides the same-domain frame reuse.
 - **default**: the parameter **defaults.isolateView** is `false` by default, and you can overwrite it using the **isolateView** value on a single node level.
 
+### tooltipText
+- **type**: string
+- **description**: allows you to add a tooltip to any of the links and buttons of the left side navigation and top navigation. You can set the parameter with the text that will appear on the tooltip. 
+- **example**:
+```javascript
+tooltipText: 'Useful links'
+```
+
+### defaults.tooltipText
+- **type**: string
+- **description**: allows you to add a tooltip to **all** the links and buttons of the left side navigation and top navigation.
+- **default**: the parameter **defaults.tooltipText** is `null` by default, and you can overwrite it using the **tooltipText** value on a single node level.
+
```

Comment:
```suggestion
```
(cc: @alexandra-simeonova) If I'm not mistaken @rafalgamon I  believe users cannot define navigation.**defaults.tooltipText**. So this field **defaults.tooltipText** is not valid.  I know that @rafalgamon might have intended to write about its default value. By default, it is inferred that if you don't set this property, you just get no tooltip 👍